helix-term/src/commands.rs

```diff
@@ -3781,6 +3892,8 @@ mode_info! {
     "P" => paste_clipboard_before,
     /// replace selections with clipboard
     "R" => replace_selections_with_clipboard,
+    /// perform code action
+    "a" => code_action,
```

Comment:
I think later we should have a lsp mode for these stuff. But we can leave this for now.